backend/routes.py

```
@@ -0,0 +1,78 @@
+from flask import Blueprint, jsonify, request
+from utils import run_simulation_step, get_simulation_state, get_all_posts
+from socket_manager import emit_agent_action
```

Ruff (0.12.2)

`socket_manager.emit_agent_action` imported but unused

Remove unused import: `socket_manager.emit_agent_action`

(F401)